<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QNotepad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>41</y>
      <width>229</width>
      <height>199</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="documentTitle">
     <string>Untitled</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Create"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Repeat"/>
    <addaction name="separator"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Delete"/>
    <addaction name="action_Cut"/>
   </widget>
   <widget class="QMenu" name="menu_Search">
    <property name="title">
     <string>&amp;Search</string>
    </property>
    <addaction name="action_Find"/>
    <addaction name="action_Find_Next"/>
    <addaction name="action_Find_Forward"/>
    <addaction name="action_Replace"/>
    <addaction name="separator"/>
    <addaction name="action_Jump_To"/>
   </widget>
   <widget class="QMenu" name="menu_Option">
    <property name="title">
     <string>&amp;Option</string>
    </property>
    <addaction name="action_Font"/>
    <addaction name="action_AutoIndent"/>
    <addaction name="action_WordWrap"/>
    <addaction name="action_LineNumber"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Search"/>
   <addaction name="menu_Option"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Create">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Repeat">
   <property name="text">
    <string>&amp;Repeat</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="text">
    <string>&amp;Find</string>
   </property>
  </action>
  <action name="action_Find_Next">
   <property name="text">
    <string>Find &amp;Next</string>
   </property>
  </action>
  <action name="action_Find_Forward">
   <property name="text">
    <string>Find &amp;Forward</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="text">
    <string>&amp;Replace</string>
   </property>
  </action>
  <action name="action_Jump_To">
   <property name="text">
    <string>&amp;Jump To</string>
   </property>
  </action>
  <action name="action_Font">
   <property name="text">
    <string>&amp;Font</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_AutoIndent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;AutoIndent</string>
   </property>
  </action>
  <action name="action_WordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;WordWrap</string>
   </property>
  </action>
  <action name="action_LineNumber">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;LineNumber</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
